README.md

```diff
@@ -1,14 +1,104 @@
-# Project
+# Microsoft Entra Wallet Library
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+## Introduction
+The Microsoft Entra Wallet Library for iOS gives your app the ability to begin using the Microsoft Identity platform by supporting the issuance and presentation of Verified Ids in accordance with OpenID Connect, Presentation Exchange, Verifiable Credentials, and more up and coming industry standards.
```

Comment:
"the Microsoft Identity platform" could be confused for other Identity SDKs (MSAL). I would recommend the more specific "Microsoft Entra Verified ID platform"